application-admintools-default/src/main/resources/templates/securityTemplate.vm

```diff
@@ -0,0 +1,60 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+
+{{html clean='false'}}
+
+#if($security['serverFound'] == 'true')
+
+  <div class="adminToolsDashboardItem">
+    <h2><span>$services.icon.renderHTML('lock')</span> <span>$escapetool.xml($services.localization.render('adminTools
+    .dashboard.section.security.title'))</span></h2>
+    <p>$escapetool.xml($services.localization.render('adminTools.dashboard.section.security.description'))</p>
+    <hr />
+    <ul>
+      <li>
+        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.activeEncoding')): $security['activeEncoding']
+      </li>
+      <li>
+        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.configurationEncoding')): $security['configurationEncoding']
+      </li>
+      <li>
+        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.PWD')): $security['PWD']
+      </li>
+      <li>
+        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.LANG')): $security['LANG']
+      </li>
+      <li>
+        $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.fileEncoding')): $security['fileEncoding']
+      </li>
+    </ul>
+  </div>
+
+
+#else
+
+  #set($warningMessage = $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.error')))
+  #warning($warningMessage)
+
+
+#end
+
+{{/html}}
+
+
```

Comment:
There are too many newlines in here. Personally, I think you need none. But if you think some separation is needed, a single newline should be used (for the other template as well)